WORKSPACE

```
@@ -90,11 +90,21 @@ http_archive(
     urls = ["https://github.com/googleapis/gapic-generator/archive/8e930b79e846b9d4876462be9dc4c1dbc04e2903.zip"],
 )
 
-# WebAssembly Binary Toolkit (forked by tiziano88).
-git_repository(
+# WebAssembly Binary Toolkit
+http_archive(
     name = "wabt",
-    commit = "30e914b1630db13080cc054b591ab5822b9b4768",
-    remote = "https://github.com/daviddrysdale/wabt",
+    build_file = "@//third_party/wabt:wabt.BUILD",
+    patches = [
+        "@//third_party/wabt:0003-Add-static-config-file.patch",
```

I think you can remove all the leading `@`.

```suggestion
        "//third_party/wabt:0003-Add-static-config-file.patch",
```

(BTW please do not apply these suggestions from the GitHub UI directly, the UI is quite confusing :) )